src/rgw/driver/rados/rgw_lc_tier.cc

```diff
@@ -1029,7 +1031,7 @@ int cloud_tier_restore(const DoutPrefixProvider *dpp, RGWRESTConn& dest_conn,
 
     void dump_xml(Formatter *f) const {
       encode_xml("Days", days, f);
-      if (tier) {
+      if (tier && tier != "") {
```

Comment:
nit: do we still need this empty string check (tier != "") as tier_v is optional string which is passed to this struct?